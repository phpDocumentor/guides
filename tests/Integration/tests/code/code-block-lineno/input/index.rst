Title
=====

..  code-block:: ruby
    :lineno-start: 10

    Some more Ruby code, with line numbering starting at 10.
