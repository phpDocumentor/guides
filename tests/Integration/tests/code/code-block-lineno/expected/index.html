<!DOCTYPE html>
<html lang="en">
<head>
    <title>Title</title>

</head>
<body>
    <div class="section" id="title">
    <h1>Title</h1>

            <pre><code class="language-ruby line-numbers" data-start="10">Some more Ruby code, with line numbering starting at 10.</code></pre>
    </div>

</body>
</html>
